config/jobs/ppc64le-cloud/builds/containerd-postsubmit.yaml

```diff
@@ -0,0 +1,22 @@
+postsubmits:
+  containerd/containerd:
```

Comment:
Heads up: This doesn't work unless we add a webhook to the `containerd/containerd` repository.

Reply:
alternate way would be to use the periodic type of job where is trigger based on the interval(like cronjob)